x-pack/plugins/case/server/routes/api/cases/configure/get_connectors.ts

```diff
@@ -36,8 +37,9 @@ export function initCaseConfigureGetActionConnector({ caseService, router }: Rou
           (action) =>
             SUPPORTED_CONNECTORS.includes(action.actionTypeId) &&
             // Need this filtering temporary to display only Case owned ServiceNow connectors
-            (action.actionTypeId !== SERVICENOW_ACTION_TYPE_ID ||
-              (action.actionTypeId === SERVICENOW_ACTION_TYPE_ID && action.config!.isCaseOwned))
+            (![SERVICENOW_ACTION_TYPE_ID, JIRA_ACTION_TYPE_ID].includes(action.actionTypeId) ||
+              ([SERVICENOW_ACTION_TYPE_ID, JIRA_ACTION_TYPE_ID].includes(action.actionTypeId) &&
+                action.config!.isCaseOwned))
```

Comment:
Huh, that's interesting you had to use `!` with `action.config`.  I guess we allow `config` to be undefined/null.  If for some reason one WAS undefined/null though, this code would crash with a null dereference.  I think changing this to the following might work: `action.config?.isCaseOwned` or `action.config?.isCaseOwned === true` to be more explicit.